powerlevel9k.zsh-theme

```diff
@@ -578,30 +578,39 @@ prompt_ip() {
 prompt_load() {
   # The load segment can have three different states
   local current_state="unknown"
+  local cores
+
+  if [[ "$OS" == "OSX" ]]; then
```

Comment:
About 10 lines below, there is already an `if` that sets OS specific values. Could you merge yours with this one?